packages/firestore/src/core/view_snapshot.ts

```diff
@@ -149,6 +149,30 @@ export class ViewSnapshot {
     readonly excludesMetadataChanges: boolean
   ) {}
 
+  /** Returns a view snapshot as if all documents in the snapshot were added. */
+  static fromSyncedDocuments(
```

Comment:
I'm not really sure what "synced" means in this context.  Is this related to the existing `syncedDocuments` in the View?  Else maybe `fromDocuments()` or `fromInitialDocuments()` ?

Reply:
I was debating between `fromSyncedDocuments` and `fromExistingDocuments`. `synced` is slightly wrong since this set of documents should also contain local edits, but I still liked it better than simply using `existing`. `initial` is much more clear though and doesn't suggest that these documents need to be in sync with the backend. I renamed it.